My Series List/My Series List/Model/SeriesModel.swift

```diff
@@ -14,19 +14,17 @@ enum Api {
     }
     
     class Series: Codable {
-        let id: Double?
-        let title: String?
-        let overView: String?
-        let posterPath: String?
-        let voteAverage: Double?
+        var id: Int32?
```

Comment:
i changed this because I needed to instantiate a Series from this class and inject some data in it and it didn't allow me because the properties were constants